scripts/patch-manifests.js

```diff
@@ -1,6 +1,5 @@
 /*
   https://github.com/octokit/webhooks-methods.js/issues/45
-
```

Comment:
any reason for all of those extra spaces? 😆

Reply:
`prettier --write`